conanfile.py

```diff
@@ -15,16 +15,23 @@ class OpenStudioBuildRecipe(ConanFile):
     generators = "CMakeDeps"  # CMakeToolchain explicitly instantiated
 
     options = {
+        "with_ruby": [True, False],
+        "with_python": [True, False],
+        "with_csharp": [True, False],
         "with_testing": [True, False],
         "with_benchmark": [True, False],
     }
     default_options = {
+        "with_ruby": True,
+        "with_python": True,
+        "with_csharp": False,
```

Comment:
New options to allow disabling ruby (won't build on win32), and tweak the cmake vars as well.